src/Framework/MockObject/Builder/InvocationMocker.php

```diff
@@ -68,7 +68,7 @@ public function id($id): self
         return $this;
     }
 
-    public function will(Stub $stub): Identity
+    public function will(Stub $stub): self
```

Comment:
See https://github.com/vimeo/psalm/issues/2102 about current failures. For now, will update baseline